src/Groups/libraries/atlasgroups.jl

```diff
@@ -109,21 +128,43 @@ julia> g = atlas_group("M11");  # Mathieu group M11
 
 julia> h1, emb = atlas_subgroup(g, 1);  h1
 Permutation group of degree 11 and order 720
+with 2 generators
+  (1,4)(2,10)(3,7)(6,9)
+  (1,6,10,7,11,3,9,2)(4,5)
 
 julia> order(h1)  # largest maximal subgroup of M11
 720
 
 julia> h2, emb = atlas_subgroup("M11", 1);  h2
 Permutation group of degree 11 and order 720
+with 2 generators
+  (1,4)(2,10)(3,7)(6,9)
+  (1,6,10,7,11,3,9,2)(4,5)
 
 julia> h3, emb = atlas_subgroup(MatrixGroup, "M11", 1 );  h3
 Matrix group of degree 10
   over prime field of characteristic 2
+with 2 generators
+  [0   1   0   1   0   1   0   0   0   0]
+  [0   0   1   0   0   0   0   0   0   0]
+  [0   1   0   0   0   0   0   0   0   0]
+  [1   1   1   1   1   0   0   0   0   0]
+  [1   1   0   1   1   1   0   0   0   0]
+  [0   1   0   1   1   0   0   0   0   0]
+  [0   0   1   1   1   1   1   0   0   0]
+  [0   0   0   1   1   0   0   1   1   0]
+  [0   0   1   1   1   1   0   0   1   0]
+  [0   0   1   0   0   1   0   0   1   1]
+
+  ⋮
 
```

Comment:
Here we can see what happens if the generators of a matrix group "do not fit": I am inserting a vertical dots (unicode). But this should be changed if unicode is not allowed -- but to what? Perhaps just horizontal `. . . ` or `...`?
```suggestion
Matrix group of degree 10
  over prime field of characteristic 2
with 2 generators
  [0   1   0   1   0   1   0   0   0   0]
  [0   0   1   0   0   0   0   0   0   0]
  [0   1   0   0   0   0   0   0   0   0]
  [1   1   1   1   1   0   0   0   0   0]
  [1   1   0   1   1   1   0   0   0   0]
  [0   1   0   1   1   0   0   0   0   0]
  [0   0   1   1   1   1   1   0   0   0]
  [0   0   0   1   1   0   0   1   1   0]
  [0   0   1   1   1   1   0   0   1   0]
  [0   0   1   0   0   1   0   0   1   1]

  ...

```

Reply:
Both are equally bad, so just go with the vertical dots. Then we are consistent with the bad solution in julia itself.

Semi-serious suggestion: print something like
```
[1 generator omitted]
```